azuredevops/internal/acceptancetests/resource_branchpolicy_test.go

```diff
@@ -5,184 +5,159 @@ package acceptancetests
-		buildValidationOptions: buildValidationPolicyOpts{false, false, "build validation rename", 720},
-	}
-
+func TestAccBranchPolicyMinReviewers_CreateAndUpdate(t *testing.T) {
 	minReviewerTfNode := "azuredevops_branch_policy_min_reviewers.p"
-	buildVlidationTfNode := "azuredevops_branch_policy_build_validation.p"
-	autoReviewerTfNode := "azuredevops_branch_policy_auto_reviewers.p"
 
 	resource.Test(t, resource.TestCase{
-		PreCheck:  func() { testutils.PreCheck(t, &[]string{"AZDO_TEST_AAD_USER_EMAIL"}) },
+		PreCheck:  func() { testutils.PreCheck(t, nil) },
 		Providers: testutils.GetProviders(),
 		Steps: []resource.TestStep{
 			{
-				Config: getHCL(opts1),
+				Config: getMinReviewersHcl(true, true, 1, false),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttrSet(minReviewerTfNode, "id"),
 					resource.TestCheckResourceAttr(minReviewerTfNode, "blocking", "true"),
 					resource.TestCheckResourceAttr(minReviewerTfNode, "enabled", "true"),
-					resource.TestCheckResourceAttr(autoReviewerTfNode, "enabled", "true"),
-					resource.TestCheckResourceAttr(autoReviewerTfNode, "blocking", "true"),
-					resource.TestCheckResourceAttr(buildVlidationTfNode, "enabled", "true"),
-					resource.TestCheckResourceAttr(buildVlidationTfNode, "enabled", "true"),
 				),
 			}, {
-				Config: getHCL(opts2),
+				Config: getMinReviewersHcl(false, false, 2, true),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttrSet(minReviewerTfNode, "id"),
 					resource.TestCheckResourceAttr(minReviewerTfNode, "blocking", "false"),
 					resource.TestCheckResourceAttr(minReviewerTfNode, "enabled", "false"),
-					resource.TestCheckResourceAttr(autoReviewerTfNode, "enabled", "false"),
-					resource.TestCheckResourceAttr(autoReviewerTfNode, "blocking", "false"),
-					resource.TestCheckResourceAttr(buildVlidationTfNode, "enabled", "false"),
-					resource.TestCheckResourceAttr(buildVlidationTfNode, "enabled", "false"),
 				),
 			}, {
 				ResourceName:      minReviewerTfNode,
 				ImportStateIdFunc: testutils.ComputeProjectQualifiedResourceImportID(minReviewerTfNode),
 				ImportState:       true,
 				ImportStateVerify: true,
-			}, {
-				ResourceName:      buildVlidationTfNode,
-				ImportStateIdFunc: testutils.ComputeProjectQualifiedResourceImportID(buildVlidationTfNode),
-				ImportState:       true,
-				ImportStateVerify: true,
-			}, {
-				ResourceName:      autoReviewerTfNode,
-				ImportStateIdFunc: testutils.ComputeProjectQualifiedResourceImportID(autoReviewerTfNode),
-				ImportState:       true,
-				ImportStateVerify: true,
 			},
 		},
 	})
 }
 
-type minReviewPolicyOpts struct {
-	enabled          bool
-	blocking         bool
-	reviewers        int
-	submitterCanVote bool
-}
+func getMinReviewersHcl(enabled bool, blocking bool, reviewers int, submitterCanVote bool) string {
```

Comment:
Can you, for consistency reasons, move this function to the `hcl.go` file?

Reply:
As by @xuzhang3 [comment here](https://github.com/terraform-providers/terraform-provider-azuredevops/pull/74/files#r454762972) I'll retract all my comments here about moving (all) HCL code to a shared file.

Reply:
@xuzhang3 We need to come up with a new system for how we want to do this